connector/connect/common/src/main/protobuf/spark/connect/relations.proto

```diff
@@ -91,8 +91,11 @@ message Unknown {}
 
 // Common metadata of all relations.
 message RelationCommon {
+  // (Required) A globally unique id for a given connect plan.
+  int64 plan_id = 1;
+
   // (Required) Shared relation metadata.
-  string source_info = 1;
+  string source_info = 2;
```

Comment:
This is a breaking change please do not do this. Simply make the `plan_id` a 2, it does not matter.

Reply:
ok